scripts/auto-rebase/assets.yaml

```diff
@@ -149,20 +149,21 @@ assets:
       - file: 0000_20_kube-apiserver-operator_00_scc-restricted.yaml
 
   - dir: controllers/route-controller-manager/
+    no_clean: True
     src: cluster-openshift-controller-manager-operator/bindata/v3.11.0/openshift-controller-manager/
     files:
       - file: informer-clusterrolebinding.yaml
       - file: informer-clusterrole.yaml
       - file: route-controller-manager-ingress-to-route-controller-clusterrolebinding.yaml
       - file: route-controller-manager-ingress-to-route-controller-clusterrole.yaml
-      - file: route-controller-manager-leader-rolebinding.yaml
-      - file: route-controller-manager-leader-role.yaml
       - file: route-controller-manager-ns.yaml
       - file: route-controller-manager-sa.yaml
       - file: route-controller-manager-separate-sa-rolebinding.yaml
       - file: route-controller-manager-separate-sa-role.yaml
       - file: route-controller-manager-tokenreview-clusterrolebinding.yaml
       - file: route-controller-manager-tokenreview-clusterrole.yaml
+      - file: route-controller-manager-authentication-reader-rolebinding.yaml
+        ignore: "required for initializing delegating authentication when starting a server"
```

Comment:
is this the correct way how to introduce a new custom RBAC?

Reply:
thanks for the explanation, hopefully it will be better now (but have not tested it...)